frontend/src/components/RestrictedArea.js

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: 2021 Dominic Heil <d.heil@campus.tu-berlin.de>, Christoph Ehm <christoph.ehmendoerfer@campus.tu-berlin.de>
+
+import React, { useEffect, useState } from "react";
+import appwriteApi from "../api/appwriteApi";
+import CenterFlexBox from "./CenterFlexBox";
+import ParagraphTypography from "./ParagraphTypography";
+import { Link } from "react-router-dom";
+import { activeTextColor } from "../assets/jss/colorPalette";
+import { arrayIntersection } from "../utils/utils";
+
+const ErrorMessage = ({ children }) => (
+	<CenterFlexBox>
+		<ParagraphTypography>{children}</ParagraphTypography>
+	</CenterFlexBox>
+);
+
+const noAccessMessage = <ErrorMessage>
+	You are trying to access a user restricted area. Please <Link to="/login" style={{ textDecorationLine: "none", color: activeTextColor }}>login</Link>.
+</ErrorMessage>;
+
+const loadingMessage = <ErrorMessage>
+	Loading...
+</ErrorMessage>;
+
+const noMemberAccessMessage = <ErrorMessage>
+	You are trying to access a restricted area. If you believe you should have access to this area, please contact an admin directly.
+</ErrorMessage>;
+
+const adminTeamName = "Admins";
```

Comment:
can we put these constants into a `config` file to reuse later?

Reply:
Good idea, I put it into `utils/config.js`.